src/node/rpc/frontend.h

```diff
@@ -530,7 +530,16 @@ namespace ccf
       }
 
       // Retrieve id of caller
-      auto caller_id = valid_caller(tx, ctx.caller_cert);
+      std::optional<CallerId> caller_id;
+      if (ctx.is_create_request)
```

Comment:
If the `create` endpoint is moved to `nodefrontend.h`, all these checks can go because the NodeFrontend does not check the identity of the client.

Reply:
discussed offline